packages/app/src/hub/HubApp.tsx

```diff
@@ -36,6 +36,17 @@ const _HubApp: ForwardRefRenderFunction<
   console.log("Hub page tree", myTree);
   const { totalCarbon } = useCarbon();
 
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      if (showHub) {
+        updateShowHubNav(true);
+      }
+    }, 5000);
+    return () => {
+      clearInterval(intervalId);
+    };
+  }, [showHub]);
```

Comment:
No good idea, whenever showHub changes you're creating an interval.
I suggest a one-time effect with a timeout, but there needs to be a way to cancel the timeout once the user clicks

Reply:
I tried, but I couldn't figure that out, so I just deleted the timeout.

Reply:
OK, I'll take a look and split the task in the issue. Thanks!